main/curl/APKBUILD

```diff
@@ -82,7 +82,7 @@ builddir="$srcdir/$pkgname-$pkgver"
 build() {
 	cd "$builddir"
 	autoreconf -vif
-	./configure \
+	./configure LDFLAGS=-static PKG_CONFIG='pkg-config --static' \
```

Comment:
It looks like this will enforce static linking of curl itself. I don't mind that we ship a static library of curl in a subpackage, but we will not build curl static for all of our users.

Reply:
No? I got:
```
ldd usr/bin/curl
        /lib/ld-musl-x86_64.so.1 (0x7f9c22c9d000)
        libssl.so.1.1 => /lib/libssl.so.1.1 (0x7f9c22b70000)
        libcrypto.so.1.1 => /lib/libcrypto.so.1.1 (0x7f9c228f2000)
        libz.so.1 => /lib/libz.so.1 (0x7f9c226db000)
        libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x7f9c22c9d000)
```